includes/frontend/class-wcmpbe-checkout.php

```diff
@@ -509,6 +509,7 @@ private static function getDeliveryOptionsConfigMap(string $carrier): array
            "carrierSettings.$carrier.allowPickupLocations"  => [WCMPBE_Settings::SETTING_CARRIER_PICKUP_ENABLED, 'isEnabled', false],
            "carrierSettings.$carrier.allowSaturdayDelivery" => [WCMPBE_Settings::SETTING_CARRIER_SATURDAY_DELIVERY_ENABLED, 'isEnabled', false],
            "carrierSettings.$carrier.allowSignature"        => [WCMPBE_Settings::SETTING_CARRIER_SIGNATURE_ENABLED, 'isEnabled', false],
+           "carrierSettings.$carrier.allowShowDeliveryDate" => [WCMPBE_Settings::SETTING_CARRIER_ALLOW_SHOW_DELIVERY_DATE, 'isEnabled', false],
```

Comment:
Nu je deze heb toegevoegd is `deliveryDaysWindow` op regel 520 dan nog wel nodig?

Reply:
Inderdaad, weggehaald.